package.json

```diff
@@ -33,19 +33,22 @@
     "eslint": "^4.19.1",
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.12.0",
+    "eth-crypto": "^1.2.4",
     "ethereumjs-wallet": "0.6.0",
     "ganache-cli": "^6.1.3",
     "path": "0.12.7",
     "solidity-coverage": "^0.5.4",
     "solium": "^1.1.7",
     "solium-plugin-zeppelin": "^0.0.2",
     "truffle": "^4.1.11",
-    "web3": "^0.18.2",
+    "truffle-hdwallet-provider": "0.0.6",
+    "truffle-solidity-loader": "^0.0.8",
```

Comment:
Do we need the `truffle-solidity-loader` package? Because npm warns that it has a critical vulnerability that cannot be automatically fixed.

Reply:
I've deleted it in https://github.com/wibsonorg/wibson-core/pull/74 and there was no problem running the tests

Reply:
It has been added as a dependency in the first commit (https://github.com/wibsonorg/wibson-core/commit/7fb71376f2372f9855e0e2ec5af153d5e3833c99#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12) but I believe it was never needed.

Reply:
Updated! 👍